src/pkg/retention/res/selectors/regexp/selector.go

```diff
@@ -15,32 +15,73 @@
 package regexp
 
 import (
+	"github.com/bmatcuk/doublestar"
```

Comment:
Note that this ***isn't*** a regex matching library, it does something slightly different and I'm not sure how applicable it is to Harbor. If we're going to use this then the selector should probably be renamed accordingly (along with documentation being updated)

Reply:
Yes, that's not regexp. We'd like to keep the same pattern framework in the whole harbor. So, I agree with the rename.